Sources/SwiftProtobuf/Enum.swift

```diff
@@ -17,7 +17,7 @@
 
 import Swift
 
-public protocol Enum: RawRepresentable, Hashable, CustomDebugStringConvertible, FieldType, MapValueType {
+public protocol Enum: RawRepresentable, Hashable, CustomDebugStringConvertible {
     init()
     init?(jsonName: String)
     init?(protoName: String)
```

Comment:
Can/Should these last two move into an internal extension?  i.e. - does the inner plumbing on the library only need to call them and do we see a need to expose them to developers?